Documentation/DocuBlocks/Rest/Administration/get_admin_server_availability.md

```diff
@@ -0,0 +1,24 @@
+
+@startDocuBlock get_admin_server_availability
+@brief Get to know whether this server is a Coordinator or DB-Server
```

Comment:
"Get to know whether this server is a Coordinator or DB-Server" --> copy / paste error? probably it is "Return whether or not a server is available"